zipline/data/data_portal.py

```diff
@@ -94,6 +94,7 @@ def __init__(self,
         self.sources_map = {}
 
         self.sim_params = sim_params
+        self.daily_mode = (self.sim_params.data_frequency == "daily")
```

Comment:
Could we use `data_frequency` here instead? Mainly for consistency, since we do that elsewhere.

Also, I was a little surprised by this result:

```
In [9]:
daily_bar = True

In [10]:
%%timeit
daily_bar == True
10000000 loops, best of 3: 52 ns per loop


In [4]:
data_frequency = 'daily'

In [7]:
%%timeit
data_frequency == 'daily'
10000000 loops, best of 3: 38.8 ns per loop
```